Documentation for configurable merge policy (#5137)
opensearch-project/documentation-website: _api-reference/cluster-api/cluster-settings.md

| cluster.persistent_tasks.allocation.recheck_interval | Time unit | The cluster manager automatically checks whether or not persistent tasks need to be assigned when the cluster state changes in a significant way. There are other factors, such as memory usage, that will affect whether or not persistent tasks are assigned to nodes but do not otherwise cause the cluster state to change. This setting defines how often assignment checks are performed in response to these factors. Default is `30 seconds`, with a minimum of `10 seconds` being required. |
| remote_store.moving_average_window_size | Integer | The moving average window size used to calculate the rolling statistic values exposed through the [Remote Store Stats API]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/remote-store/remote-store-stats-api/). Default is `20`. Minimum enforced is `5`. |

> | indices.time_series_index.default_index_merge_policy | String | This setting allows to specify the default merge policy for time-series indexes, particularly those with a `@timestamp` field such as data streams. The two available options are: `tiered` (default) and `log_byte_size`. `log_byte_size` is recommended for time-series indexes to enhance performance on range queries on `@timestamp` field. To override the merge policy on a per-index basis, the `index.merge.policy` index setting can be used. |

Reviewer: ```suggestion
| indices.time_series_index.default_index_merge_policy | String | This setting allows you to specify the default merge policy for time-series indexes, particularly for those with an `@timestamp` field such as data streams. The two available options are: `tiered` (default) and `log_byte_size`.  We recommend `log_byte_size` for time-series indexes to enhance the performance of range queries with the `@timestamp` field. To override the merge policy on a per-index basis, you can use the `index.merge.policy` index setting. |
```
